src/index.html

```diff
@@ -6,8 +6,60 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div
+      class="product"
+      data-qa="card"
+    >
+      <img
+        class="product__image"
+        src="./images/imac.jpeg"
+        alt="product__image"
+      >
+
+      <div class="product__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product__code">
+        Product code: 195434
+      </div>
+
+      <div class="stars">
+        <div class="stars__block">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="stars__review">Reviews: 5</p>
+      </div>
+
```

Comment:
```suggestion
      <div class="card__review">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p class="stars__review">Reviews: 5</p>
      </div>

```

Reply:
I cant change stars name, because thats the name of the parent element my design is based on (((( but other suggestions were changed